src/_common/notification/notification-model.ts

```diff
@@ -423,6 +434,19 @@ export function getNotificationText(notification: Notification, plaintext = fals
 			return gameTitle + postTitle;
 		}
 
+		case Notification.TYPE_POST_FEATURED_IN_COMMUNITY: {
+			const postCommunity = notification.action_model as FiresidePostCommunity;
+
+			return _process(
+				Translate.$gettextInterpolate(
+					`Your post in <em>%{ community }</em> community got featured!`,
```

Comment:
I think it should be "Your post in the whatever community has been featured".

Reply:
yup. i was messing around so much with how to keep this translated properly i forgot how to english.